src/include/network/routines.rb

```diff
@@ -742,24 +756,62 @@ def SetIfaceDown(dev_name)
 
     # Tries to set all interfaces up
     #
-    # @return [boolean] false if some of interfaces cannot be set up
+    # @return [boolean] false if interfaces are empty or some
+    # interface cannot be bring up
```

Comment:
Sorry, but this is plain wrong. This is not how one handles the empty set. And you would be able to see it if there was an actual use of this API, which there isn't.